migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,314 @@
+-- Enums
+
+CREATE TYPE gender                  AS ENUM ('male', 'female');
+CREATE TYPE role                    AS ENUM ('user', 'admin');
+CREATE TYPE status                  AS ENUM ('available', 'unavailable');
+CREATE TYPE association             AS ENUM ('sponsor', 'organization', 'media');
+CREATE TYPE task_priority           AS ENUM ('low', 'medium', 'high');
+CREATE TYPE acceptance_status       AS ENUM ('pending', 'accepted', 'rejected');
+CREATE TYPE employee_level          AS ENUM ('basic', 'organizer', 'manager');
```

Comment:
This also isn't right. It should be 'basic', 'manager' and 'upper manager' as it is in the erd.